frontend/src/app/plans/account/page.tsx

```diff
@@ -0,0 +1,85 @@
+import Image from "next/image";
+import { notFound } from "next/navigation";
+import React from "react";
+
+import { Separator } from "@/components/ui/separator";
+import { createUsosService } from "@/lib/usos";
+
+const USER_STATUSES = {
+  0: "Niegdy nie byłeś studentem",
+  1: "Były Student",
+  2: "Aktywny Student",
+};
+
+const USER_SEX = {
+  M: "Mężczyzna",
+  K: "Kobieta",
+};
+
+export default async function ProfilePage() {
+  let profile;
+
+  try {
+    const usos = await createUsosService();
+    profile = await usos.getProfile();
+  } catch {
+    return notFound();
+  }
+
+  const sex = USER_SEX[profile.sex as keyof typeof USER_SEX] || "Nieznana";
+
+  return (
+    <div className="space-y-6">
+      <div>
+        <h3 className="text-lg font-medium">Profil</h3>
+        <p className="text-sm text-muted-foreground">
+          Są to oficjalne dane z USOSweb i możesz je zmienić właśnie tam.
+        </p>
+      </div>
+      <Separator />
+      <div className="flex items-start gap-4 rounded-md border p-5">
+        <div className="size-[51px] min-w-[51px] rounded-full border">
+          <Image
+            src={profile.photo_urls["50x50"]}
+            alt={""}
+            width={50}
+            height={50}
+            unoptimized
+            className="size-[50px] rounded-full"
+          />
+        </div>
+        <div className="flex w-full flex-col">
+          <h1 className="text-lg font-semibold">
+            {profile.first_name} {profile.last_name}
+          </h1>
+          <div className="mt-2 w-full">
+            <div className="flex w-full items-center justify-between">
+              <p>Adres e-mail:</p>
+              <h3 className="font-medium">
+                {profile.student_number}@student.pwr.edu.pl
+              </h3>
+            </div>
+            <div className="flex w-full items-center justify-between">
+              <p>Płeć:</p>
+              <h3 className="font-medium">{sex}</h3>
+            </div>
```

Comment:
imo te dane typu płeć czy to że jesteś studenciakiem są niepotrzebne, jeszcze jakaś julka sie przyczepi

Reply:
szczerze, i dont care, ladnie wyglada jak masz dane, poza tym to usos przyjmuje k m lub inne wiec no, not my fault